app/components/file-uploader.js

```diff
@@ -151,99 +155,78 @@ export default Component.extend(Analytics, {
-            // and then uploads file to newly created component.
+            // Upload case for using a new component and a new file for the preprint.
+            // Creates component of parent node and then uploads file to newly created component.
             this.get('metrics')
                 .trackEvent({
                     category: 'button',
                     action: 'click',
-                    label: 'Submit - Save and Continue, New Component New File'
+                    label: 'Submit - Save and Continue, New Component New File',
                 });
-            let node = this.get('node');
+            const node = this.get('node');
             node
                 .addChild(this.get('title'))
-                .then(child => {
-                    this.set('parentNode', node);
-                    this.set('node', child);
-                    this.set('basicsAbstract', this.get('node.description') || null);
-                    this.send('upload');
-                    this.set('newNode', true);
-                    this.set('applyLicense', true);
-                })
-                .catch(() => {
-                    this.set('uploadInProgress', false);
-                    this.get('toast').error(this.get('i18n').t('components.file-uploader.could_not_create_component'));
-                });
+                .then(this._createComponent.bind(this))
+                .catch(this._failCreateComponent.bind(this));
         },
 
         uploadFileToExistingNode() {
-            // Upload case for using an existing node with a new file for the preprint.  Updates title of existing node and then uploads file to node.
+            // Upload case for using an existing node with a new file for the preprint.
+            // Updates title of existing node and then uploads file to node.
             // Also applicable in edit mode.
             this.get('metrics')
                 .trackEvent({
                     category: 'button',
                     action: 'click',
-                    label: `${this.get('editMode') ? 'Edit' : 'Submit'} - Save and Continue, ${this.get('nodeLocked') ? 'Save File/Title Edits' : 'Existing Node New File'}`
+                    label: `${this.get('editMode') ? 'Edit' : 'Submit'} - Save and Continue, ${this.get('nodeLocked') ? 'Save File/Title Edits' : 'Existing Node New File'}`,
                 });
             if (this.get('nodeLocked')) { // Edit mode
                 this.set('uploadInProgress', true);
             }
 
-            let model = this.get('model');
-            let node = this.get('node');
+            const model = this.get('model');
+            const node = this.get('node');
             this.set('basicsAbstract', this.get('model.description') || null);
-            let currentNodeTitle = node.get('title');
+            const currentNodeTitle = node.get('title');
+            this.set('currentNodeTitle', currentNodeTitle);
```

Comment:
Might need to look into this, not sure why this was added... might have been a regress